client/web/src/repo/commits/GitCommitNode.tsx

```diff
@@ -94,102 +102,162 @@ export const GitCommitNode: React.FunctionComponent<GitCommitNodeProps> = ({
             )}
         </div>
     )
+
+    const commitMessageBody =
+        expandCommitMessageBody || showCommitMessageBody ? (
+            <div className="w-100">
+                <pre className="git-commit-node__message-body">{node.body}</pre>
+            </div>
+        ) : undefined
+
+    const bylineElement = (
+        <GitCommitNodeByline
+            className="text-muted git-commit-node__byline"
+            author={node.author}
+            committer={node.committer}
+            compact={Boolean(compact)}
```

Comment:
is compact not already a boolean / not a required prop?

```suggestion
            compact={compact}
```

Reply:
Since this is an old implementation I prefer not to modify so much the original code. I'll put a TODO comment to review at some point in the future.